### ISSUE TITLE
Add documentation on minimal working example of an IdP

### ISSUE THREAD

**Issue by skoranda** [IdentityPython/pysaml2#725]
The documentation should cover how to create the IdP from configuration, process an authentication request, and send a signed assertion/response.

**Comment by peppelinux** [comment]
This would be good point to start from
https://github.com/UniversitaDellaCalabria/uniAuth/blob/004a4b50730990c625b23e7a4d4961ed40127d3b/uniauth/urls.py#L7

Each resource matches to those functions that handles persistence (SAMLrequests and generally session management) and process requests following those until the response happens. A general I/O schema the explain which pysaml2 internals would be involved each time, where and how, which argomenta would be carried in the flow

**Comment by bjm88** [comment]
yes this is much needed.  The example idp is confusing to follow, just need to know how to configure IDP with key/cert to give metadata, configure an SP to send response to, configure Identity attributes to pass.  No clear 1, 2, 3 step docs for this.
